### ISSUE TITLE
Can not connect to coordinator Web UI

### ISSUE THREAD

**Issue by Jenar78** [arangodb/arangodb#2171]
## my environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [ ] 2.8
- [x] 3.1
- [ ] self-compiled devel branch

On this operating system:
- [ ] DCOS on
  - [ ] AWS
  - [ ] Azure
  - [ ] own infrastructure
- [ ] Linux 
  - [ ] Debian .deb
  - [ ] Ubuntu .deb
  - [ ] SUSE   .rpm
  - [ ] RedHat .rpm
  - [ ] Fedora .rpm
  - [ ] Gentoo
  - [ ] docker - official docker library
  - [ ] other:
- [x] Windows, version: 10x64
![arangodb-cluster](https://cloud.githubusercontent.com/assets/23477162/20305445/f4e2224c-ab35-11e6-90ee-5b6f793a760a.PNG)

- [ ] MacOS, version:

### this is a web interface-related issue:
I'm using the web interface with this browser:     running on this OS:
- [ ] authentication is enabled?
- [ ] using the cluster?
- [ ] _system database?

These are the steps to reproduce:
i am trying to setup some local test as described in https://docs.arangodb.com/3.1/Manual/Deployment/Local.html
so i have installed last version of ArangoDB 3.1.
and used attached scripts to start local cluster.
 
But after i start coordinator it stays at „INFO In database ‚_system‘ : state coordinator…“ (..is ready for business. Have fun! Doesn’t come?)

The following problem occurs: [Screenshot?] 
I can’t connect to coordinator Web UI.

01-startLocalCluster-Agent.bat:
```
SET ARANGOD="C:\Program Files\ArangoDB3 3.1.0\usr\bin\arangod.exe"
ECHO "Starting Agent..."
%ARANGOD% --server.endpoint tcp://0.0.0.0:5001 --server.authentication false --agency..activate true --agency.size 1 --agency.endpoint tcp://127.0.0.1:5001 --agency.supervision true agency1 
```
 
02-startLocalCluster-Database1.bat
```
SET ARANGOD="C:\Program Files\ArangoDB3 3.1.0\usr\bin\arangod.exe"
ECHO "Starting Database1..."
%ARANGOD% --server.authentication false --server.endpoint tcp://0.0.0.0:8529 --cluster.my-address tcp://127.0.0.1:8529 --cluster.my-local-info db1 --cluster.my-role PRIMARY --cluster.agency-endpoint tcp://127.0.0.1:5001 primary1
```
 
03-startLocalCluster-Database2.bat
```
SET ARANGOD="C:\Program Files\ArangoDB3 3.1.0\usr\bin\arangod.exe"
ECHO "Starting Database2..."
%ARANGOD% --server.authentication false --server.endpoint tcp://0.0.0.0:8530 --cluster.my-address tcp://127.0.0.1:8530 --cluster.my-local-info db2 --cluster.my-role PRIMARY --cluster.agency-endpoint tcp://127.0.0.1:5001 primary2
```
 
04-startLocalCluster- Coordinator.bat
```
SET ARANGOD="C:\Program Files\ArangoDB3 3.1.0\usr\bin\arangod.exe"
ECHO "Starting Coordinator"
%ARANGOD% --server.authentication false --server.endpoint tcp://0.0.0.0:8531 --cluster.my-address tcp://127.0.0.1:8531 --cluster.my-local-info coord1 --cluster.my-role COORDINATOR --cluster.agency-endpoint tcp://127.0..0.1:5001 coordinator
```

**Comment by kvahed** [comment]
Thanks for the report. I'm looking into this.

**Comment by kvahed** [comment]
Fixed in 3.1.2

**Comment by Jenar78** [comment]
Great, already tested. Working without any problems,
Thank you for quick fixing it and your time.
